File: Programmers/level1/secret.js
const n = 6
let arr1 = [46, 33, 33 ,22, 31, 50]
let arr2 = [27 ,56, 19, 14, 14, 10]

let result = []
let row = ''
console.log(result)
for(let i=0;i<n;i++){
    row = (arr1[i]|arr2[i]).toString(2).replace(/0/g,' ').replace(/1/g,'#')
    console.log(row)
    while(row.length!=n){
        row = ' ' + row
    }
    result.push(row)
}
console.log(result)

File: Programmers/level1/secret.py
n = int(input())

arr1 = list(map(int,input().split()))
arr2 = list(map(int,input().split()))

arr3 = []
arr4 = []
result = []
for i in arr1:
    arr3.append(format(i,'b').zfill(n))

for i in arr2:
    arr4.append(format(i,'b').zfill(n))

for i in range(n):
    row = ''
    for j in range(n):
        if arr3[i][j] == '0' and arr4[i][j] == '0':
            row += ' '
        else:
            row +='#'
    result.append(row)

print(result)

# for x in
